en/compatibility.md

```diff
@@ -11,6 +11,11 @@ th, td { text-align: center; }
 h1, h2, h3, h4, h5, h6 { text-align: center; }
 </style>
 
+The compatibility dashboard displays evaluation of different Bitcoin wallets
```

Comment:
- it looks like "evaluation" here should be "evaluations" or "an evaluation"

- perhaps s/different/various/

- s/wallets/wallets'/